steps/qubo.py

```diff
@@ -9,4 +19,25 @@ def generate_random_qubo(size: int, seed: int = None):
         seed: seed used for RNG
     """
     qubo = generators.uniform(size, "BINARY", low=-1, high=1, seed=seed)
-    save_qubo(qubo, "qubo.json")
\ No newline at end of file
+    save_qubo(qubo, "qubo.json")
+
+
+def get_qubo_hamiltonian(qubo):
```

Comment:
I think we should indicate somehow what we return here. Maybe we could rename this to `qubo_to_openfermion_hamiltonian`?

Reply:
Done!